specs/0118-notification-center.md

```diff
@@ -0,0 +1,119 @@
+- Feature Name: Notification Center
+- Created: 2024-03-12
+- Associated Issue: 
+
+# Summary
+
+Introduce a Notification Center feature to make Pontoon the best tool to send communications to localizers directly, allowing sending of targeted in-app notifications and emails within Pontoon itself.
+
+# Motivation
+
+Currently, we can only send in-app notifications with limited targeting capabilities (selected locales within a single product). For any other communications targeting a subset of users we need to rely on engineers to extract email addresses and send emails to the targeted recipients. This feature would allow administrators of a Pontoon instance managing localization projects to send notifications or emails to targeted sets of recipients with more refined criteria such as locales, projects, roles, recency of activity, and level of activity.
+
+# Feature explanation
+
+## Notification Center
```

Comment:
Where in the UI do I access the Notification Center?

Reply:
Very good callout - I'll need to think of this.

Reply:
`Notifications` in the top menu, since we're going to get rid of the notifications tabs in project page?

Reply:
Changed to `Messaging Center`